source/common/http/http2/metadata_interface.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <unordered_map>
+
+namespace Envoy {
+namespace Http {
+namespace Http2 {
+
+/**
+ * Please refer to #2394 or go/envoy-metadata for more info about Envoy METADATA.
```

Comment:
go/envoy-metadata I think is only visible within google. :)

Can you update this comment to point to public GH issues only?

Reply:
Fixed.